tests/virtual_machines_test.go

```diff
@@ -425,7 +425,7 @@ var _ = Describe("Virtual Machines", func() {
 					}
 					return err
 
```

Comment:
How about we export `timeout+getVmFailCleanupWaitTime()` to variable with the name
totalTimeout?
its easier to debug when a function is called in separate line

Reply:
not sure I got what you mean by export. can you elaborate?

Reply:
I mean to create another variable :sweat_smile: 
`totalTime := timeout+getVmFailCleanupWaitTime()`

Reply:
ack